parcel/Example_Frostenberg_Immersion_Freezing.jl

```diff
@@ -138,18 +138,20 @@ end
 
 # Plotting
 fig = MK.Figure(size = (1200, 900))
-ax1 = MK.Axis(fig[1, 1], ylabel = "Ice Supersaturation [-]")
-ax2 = MK.Axis(fig[1, 2], ylabel = "T [K]")
-ax3 = MK.Axis(fig[2, 1], ylabel = "q_ice [g/kg]")
-ax4 = MK.Axis(fig[2, 2], ylabel = "q_liq [g/kg]")
-ax5 = MK.Axis(fig[3, 1], xlabel = "Time [min]", ylabel = "N_ice [1/cm3]")
-ax6 = MK.Axis(fig[3, 2], xlabel = "Time [min]", ylabel = "N_liq [1/cm3]")
-ax7 = MK.Axis(
-    fig[4, 1:2],
-    xlabel = "T [K]",
-    ylabel = "INPC [1/m3]",
-    yscale = log10,
-)
+kwargs = (; xgridvisible = false, ygridvisible = false,)
```

Comment:
Not sure I'm a fan of the temperature going backwards in time. But if you think it's better, lets keep it :)